prov/sockets/src/sock_cntr.c

```diff
@@ -204,12 +204,12 @@ void sock_cntr_check_trigger_list(struct sock_cntr *cntr)
 	fastlock_release(&cntr->trigger_lock);
 }
 
-static uint64_t sock_cntr_read(struct fid_cntr *cntr)
+static uint64_t sock_cntr_read(struct fid_cntr *fid_cntr)
 {
-	struct sock_cntr *_cntr;
-	_cntr = container_of(cntr, struct sock_cntr, cntr_fid);
-	sock_cntr_progress(_cntr);
-	return atomic_get(&_cntr->value);
+	struct sock_cntr *cntr;
```

Comment:
May be combine them
`struct sock_cntr *cntr = container_of(fid_cntr, struct sock_cntr, cntr_fid);`